doc/source/science_guide/sg_dynamics.rst

```diff
@@ -93,18 +94,20 @@ For clarity, the two components of Equation :eq:`vpmom` are
      -C_bv-mfu - mg{\partial H_\circ\over\partial y}. \end{aligned}
    :label: momsys
 
+On the B grid, the equations above are solved at the U point for the collocated u and v components (see figure :ref:`fig-Bgrid`). On the C grid, however, the two components are not collocated: the u component is at the E point while the v component is at the N point.
 
-A bilinear discretization is used for the stress terms
+The B grid spatial discretization is based on a variational method described in :cite:`Hunke02`. A bilinear discretization is used for the stress terms
 :math:`\partial\sigma_{ij}/\partial x_j`,
 which enables the discrete equations to be derived from the
 continuous equations written in curvilinear coordinates. In this
 manner, metric terms associated with the curvature of the grid are
-incorporated into the discretization explicitly. Details pertaining to
-the spatial discretization are found in :cite:`Hunke02`.
```

Comment:
Was this statement moved somewhere else?  We should keep it.

Reply:
This has been moved to the intro of Momentum. We now have:

The B grid spatial discretization is based on a variational method described in :cite:`Hunke02`. A bilinear discretization is used for the stress terms
:math:`\partial\sigma_{ij}/\partial x_j`,
which enables the discrete equations to be derived from the
continuous equations written in curvilinear coordinates. In this
manner, metric terms associated with the curvature of the grid are
incorporated into the discretization explicitly.

On the C grid, however, a finite difference approach is used for the spatial discretization. The C grid discretization is based on :cite:`Bouillon09, Bouillon13, Kimmritz16`.

Reply:
The first part isn't quite right -- the variational method originated in Hunke and Dukowicz 1997, and the bilinear approximation was added in the 2002 paper. The sentence that I want to keep is "Details pertaining..." because the written-out discretizations are very helpful!

Reply:
Ok corrected. Let me know if you want other modifications.